homeassistant/components/sensor/mvglive.py

```diff
@@ -0,0 +1,160 @@
+"""
+Support for real-time departure information for public transport in Munich.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.mvglive/
+"""
+
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+# A typo in the file name of the PyPI version prevents installation from PyPI
```

Comment:
Make sure to open an issue for this so we can migrate to use the PyPi version in the future.